lua/lint/linters/eugene.lua

```diff
@@ -0,0 +1,31 @@
+return {
+  cmd = 'eugene',
+  args = {
+    'lint',
+    '--format=json',
+  },
+  stdin = false,
+  parser = function(output, _)
+    local diagnostics = {}
+    if #output > 0 then
+      local ok, decoded = pcall(vim.json.decode, output)
+      if not ok then
+        return diagnostics
+      end
```

Comment:
```suggestion
      local decoded = vim.json.decode(output)
```

Decode errors would be caught further up in the framework and the error gets reported as diagnostic in the first line.

That can help catch configuration errors.